### ISSUE TITLE
Various documentation fixes

### ISSUE THREAD

**Issue by picimako** [citrusframework/citrus#964]
Hi!

I've been reading through the Citrus documentation and I've found a handful of small typos and grammatical issues, as well as some minor issues with code snippets.

I've already pushed some of the changes to my fork of Citrus, and I'd be glad to open a PR for that.

I have just one question: if you have no objections, would you like me to open one PR for the typos and another one for the code snippet?

**Comment by bbortt** [comment]
sure, go on! 😉

**Comment by christophd** [comment]
@picimako yes, please! this would be very appreciated! If you like open two separate PRs, just what suits best for you.

Recently, I have been working on code samples in documentation, too. So please be sure to update to the latest version before opening the PR, thx. 

Also in case your contribution is targeting Citrus 3.x please open the PR towards the respective [3.x](https://github.com/citrusframework/citrus/tree/3.x) branch. Otherwise you can use the main branch that represents Citrus 4.x

Many thanks!

**Comment by picimako** [comment]
One PR was more convenient, so I've opened https://github.com/citrusframework/citrus/pull/966 against `main`. :)

**Comment by christophd** [comment]
Fixed with #966
